templates/checkout/thankyou.php

```diff
@@ -22,7 +22,9 @@
 
 <div class="woocommerce-order">
 
-	<?php if ( $order ) : ?>
+	<?php if ( $order ) : 
+
+	do_action( 'woocommerce_before_thankyou', $order->id ); ?>
```

Comment:
Incorrect indentation, also should use `$order->get_id()` instead.

Reply:
Done, thanks